### PR TITLE
Feat/npm ol

### PR DESCRIPTION
**Pull request by chmelevskij**
* remove open layers map files from repo
* show map directly in the main app instead of nwjs `webview`

This should make it easier updated `ol` in the future and it's one more step closer to PWA

### CODE REVIEW

**Comment by nerdCopter** [comment]
i cannot test without GPS; need @ctzsnooze to review

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Pull request merged**
